CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [7.4.1] HotFix & Stable Release
+### Fixed Issues
+- Reverted [#1025](https://github.com/Microsoft/mssql-jdbc/pull/1025) as it contains breaking changes. This removes `hashCode()` and `equals()` APIs from `SQLServerDataTable` and `SQLServerDataColumn`.
```

Comment:
Reverted [#1025](https://github.com/Microsoft/mssql-jdbc/pull/1025) as it changes API definition for `SQLServerDataTable.getIterator()` that causes breaking change. This change also removes `hashCode()` and `equals()` APIs from `SQLServerDataTable` and `SQLServerDataColumn`.